src/pages/game/is-guessed-toast/isGuessedToast.tsx

```diff
@@ -0,0 +1,28 @@
+import { Alert, Snackbar } from '@mui/material';
+
+import { useAppSelector } from '../../../app/hooks';
+import { selectGuessedWords } from './isGuessedToastSlice';
+
+type PropsType = {
+    autoHideDuration: number;
+    onToastClose: () => void;
+};
+
+const AlreadyGuessedToast = ({ autoHideDuration, onToastClose }: PropsType) => {
```

Comment:
please rename the file to match the component name AlreadyGiessedToast

Reply:
Fixed.